crates/bitwarden/src/mobile/vault/client_ciphers.rs

```diff
@@ -0,0 +1,25 @@
+use crate::{crypto::Encryptable, error::Result, Client};
+
+use super::{client_vault::ClientVault, CipherEncryptRequest, CipherEncryptResponse};
+
+pub struct ClientCiphers<'a> {
+    pub(crate) client: &'a Client,
+}
+
+impl<'a> ClientCiphers<'a> {
+    pub async fn encrypt(&self, req: CipherEncryptRequest) -> Result<CipherEncryptResponse> {
+        let enc = self.client.get_encryption_settings()?;
+
+        let cipher = req.cipher.encrypt(enc, &None)?;
```

Comment:
Ciphers can belong to an organization, although they contain the organization_id themselves if they do.